mysql-db-init/README.md

```diff
@@ -59,26 +61,23 @@ docker run --rm -it --name mysql-db-init \
 connect to MYSQL instance
 
 ```console
-docker exec -it mysql mysql -h <local-ip> -uroot
+docker exec -it mysql mysql -h <local-ip> -uroot -pmysql
```

Comment:
As mentioned above, if you keep the `MYSQL_SUPER_USER` variable, it would make sense to use it here instead of `-uroot`.

Reply:
you should also remove the container after use with `--rm`